### PR TITLE
EverlastingCash Web3 Grant Application

### PR DESCRIPTION
**Pull request by steve1588**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A BTC or Ethereum (DAI) address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://gist.github.com/Noc2/75bc58e8ce9b5d419ff883b0cf2b8c19).
- [x] The total funding amount of the project is below USD $30k for initial grant applications and $100k for follow-up grants.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once we've successfully [delivered](https://github.com/w3f/Grant-Milestone-Delivery) the first milestone.

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. In general it looks like an interesting project and as far as I know, we don’t have anyone working on a hybrid of a collateralized and an algorithmic stablecoin at the moment. I have a few questions/comments:

- Could you remove everything related to the deployment of your network/token from the deliverables, for example, “Rococo parachain deployment”? 
- Also the deliverables of milestone 1 are not something we would support via the grants program directly. We completely understand that you need to set up your own chain and potentially learn more about how to do this, but this is not something we would fund directly. But you could integrate the costs simply into milestone 2. We are generally interested in supporting open source projects and code that are potentially useful for others in our ecosystem. 
- Just out of curiosity: Are you aware of potentially other similar (ethereum) projects? 
- What do you mean with isomorphic parachain?

**Comment by steve1588** [comment]
**Thank you very much for your review and guidance, please kindly see our replies as follows:**
> 
> * Could you remove everything related to the deployment of your network/token from the deliverables, for example, “Rococo parachain deployment”?
<p>  Yes for sure. We will remove everything related to the deployment from the deliverables of the proposal.

> * Also the deliverables of milestone 1 are not something we would support via the grants program directly. We completely understand that you need to set up your own chain and potentially learn more about how to do this, but this is not something we would fund directly. But you could integrate the costs simply into milestone 2. We are generally interested in supporting open source projects and code that are potentially useful for others in our ecosystem.
<p>  Thank you for your guidance, truly appreciated. We will adjust the development plans and revise the milestones, as you suggested.

> * Just out of curiosity: Are you aware of potentially other similar (ethereum) projects?
<p>  Good question, we have done our research and we haven’t found any project that serves exact the same purpose as ours. In terms of similar stablecoins, there are the likes of MakerDao, for example, which are crypto-collateralized, or algorithmic stablecoins like AMPL and Basis, but there is no such stablecoin like ELC that combines reserves with the algorithmic features. We believe it is an innovative product and would like to be an active part of the Polkadot ecosystem. 

> * What do you mean with isomorphic parachain?
<p>  By “isomorphic parachain”, What we wanted to say is "a parachain using shared security”.  and we will update accordingly.

<p>  Thank you again for your comments. We look forward to the good news and can’t wait to contribute to the Polkadot ecosystem by making ELC and Cycan an outstanding project.

**Comment by Noc2** [comment]
Thanks for the update. Looks good to me. Two minor issues: We don’t pay teams in DOT or KSM. Could you provide an ethereum address for a payment in DAI? (Alternatively we could also pay you in BTC). And could you update the following: “that are equivalent to $2,000 including DOT,KSM,BTC,ETH,DAI)”

**Comment by steve1588** [comment]
> Thanks for the update. Looks good to me. Two minor issues: We don’t pay teams in DOT or KSM. Could you provide an ethereum address for a payment in DAI? (Alternatively we could also pay you in BTC). And could you update the following: “that are equivalent to $2,000 including DOT,KSM,BTC,ETH,DAI)”

* For sure! Thank you so much for your approval, that means a lot to us. Dai works! We will update the payment methods accordingly and will add our Ethereum address immediately. Thank you again for the guidance.

**Review by Noc2 — DISMISSED** [review]
Looks good to me. I will share it with the rest of the team.

**Review by BenWhiteJam — DISMISSED** [review]

File: applications/EverlastingCash.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+Lastly, the team has a consolidated architectural and devops experience both in the context of experimental and in real-world large-scale deployments.
+
+
+### Team Code Repos
+* ELP-Runtime-node: https://github.com/CycanTech/ELP-Runtime-node 
+* ELC-contracts: https://github.com/CycanTech/ELC 
+
+### Team LinkedIn Profiles
+* Provided in private through Google Form.
```
Comment (alxs) [inline applications/EverlastingCash.md:154]: Which Google Form are you referring to? We don't have one for the Open Grants Program, and I also can't find a submission with your details in the General Grants one. If possible integrate your profiles here for transparency.

Comment (steve1588) [reply applications/EverlastingCash.md:154]: > 您指的是哪种Google表单？我们没有开放式赠款计划的赠款，我也无法在“普通赠款”中找到包含您的详细信息的提交。如果可能，请在此处集成您的配置文件以提高透明度。

*OK, if necessary, we will directly write the Linkedin address of the core team members in the material. We will complete this update soon, thank you.

Comment (steve1588) [reply applications/EverlastingCash.md:154]: > Which Google Form are you referring to? We don't have one for the Open Grants Program, and I also can't find a submission with your details in the General Grants one. If possible integrate your profiles here for transparency.

OK, we will directly write the Linkedin addresses of the core team members in the application, thank you.

**Review by alxs — APPROVED** [review]
Looks good! Interesting proposal, looking forward to your first delivery.

**Review by Noc2 — APPROVED** [review]

**Review by BenWhiteJam — APPROVED** [review]

**Pull request merged**